spearhead/assets/sass/_text.scss

```diff
@@ -8,6 +8,12 @@
     }
 }
 
+.wp-block a, // Target the editor link styles also.
 .entry-content a {
     font-weight: var(--global--link-font-weight);
+    border-color: var( --global--color-primary ); // Links inside the post content must be underlined.
+
+    &:hover:not(.wp-block-button__link) {
```

Comment:
I think you need to make this apply to the file block's button too: 

![Kapture 2020-12-04 at 09 36 08](https://user-images.githubusercontent.com/1202812/101176201-2a308200-3614-11eb-99e8-faed6b10cee4.gif)

Reply:
(Or specifically set the border to `0` in the file block's styles)